src/pubtools/_marketplacesvm/cloud_providers/aws.py

```diff
@@ -36,6 +36,13 @@ def get_2_digits(version: str) -> str:
         v = version.split(".")[:2]
         return ".".join(v)
 
+    def format_version(version: str, format_template: str) -> str:
+        v = version.split(".")
+        major = v[0]
+        minor = v[1]
+        patch = v[2] or 0
```

Comment:
That could fail if version is just X.Y
```
>>> version = "1.2"
>>> v = version.split(".")
>>> patch = v[2] or 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```

we might use 
```
major = v.pop(0) if v else None
minor = v.pop(0) if v else None
patch = v.pop(0) if v else None
```

Reply:
Updated this to be more resilient :)

Reply:
TYVM